generators/app/steps/CustomizableSteps/constants.js

```diff
@@ -69,8 +69,8 @@ module.exports.DEPENDENCIES = [
   'react-hook-form@^6.12.2'
 ];
 
+// TODO: What to do here about dev dependencies when using customizable, why is plop here? shouldn't it be on generators/app/constants.js ?
```

Comment:
I think we are always using customizable by default for now.
After this remove the comment.